stdlib/REPL/src/docview.jl

```diff
@@ -221,7 +238,7 @@ function lookup_doc(ex)
         str = string(ex)
         isdotted = startswith(str, ".")
         if endswith(str, "=") && Base.operator_precedence(ex) == Base.prec_assignment && ex !== :(:=)
-            op = str[1:end-1]
+            op = str[1:prevind(str, end, 1)]
```

Comment:
already fixed, so this is now causing a merge conflict
```suggestion
            op = str[1:end-1]
```